packages/iTwinUI-react/src/core/Slider/Slider.tsx

```diff
@@ -405,9 +428,10 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
         return (
           <div className='iui-slider-ticks'>
             {tickLabels.map((label, index) => (
-              <span key={index} className='iui-slider-tick'>
-                {label}
-              </span>
+              <div key={index} className='iui-slider-tick'>
```

Comment:
This change needs to be reverted

Reply:
**[DONE]** Ohh, right, thanks for reminding